src/core/__tests__/ReactMultiChildText-test.js

```diff
@@ -16,10 +16,13 @@
 require('mock-modules');
 
 var React = require('React');
+var ReactFragment = require('ReactFragment');
 var ReactTestUtils = require('ReactTestUtils');
 
 var reactComponentExpect = require('reactComponentExpect');
 
+var frag = ReactFragment.create;
```

Comment:
nit: any reason for inconsistency between this assignment and (in other tests) this:

```
function frag(obj) {
  return ReactFragment.create(obj);
}
```

?